packages/common/src/models/Analytics.ts

```diff
@@ -84,6 +84,13 @@ export enum Name {
   AUDIUS_OAUTH_COMPLETE = 'Audius Oauth: Login (authenticate) Success',
   AUDIUS_OAUTH_ERROR = 'Audius Oauth: Login (authenticate) Failed',
 
+  // Developer app
+  DEVELOPER_APP_DELETE_ERROR = 'Developer Apps: Delete app error',
```

Comment:
nit but imo order create before delete, and then order each as submit success error